<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimFilmDialog</class>
 <widget class="QewSimpleDialog" name="SimFilmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimFilmDialog</string>
  </property>
  <property name="toolTip">
   <string>The current time</string>
  </property>
  <property name="whatsThis">
   <string>This dialog can embebed other dialogs in a frame.
It can be inherited for subclassing.</string>
  </property>
  <property name="containerName">
   <cstring>QewContainerFrame</cstring>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="QewContainerFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Resolution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="resolutionComboBox">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <item>
           <property name="text">
            <string>Native</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>800x600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1024x768</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Sequence folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="folderLineEdit">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="folderToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Image format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="formatComboBox">
          <item>
           <property name="text">
            <string>PNG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>JPG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TIFF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>BMP</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="embeddedRadioButton">
          <property name="text">
           <string>Embedded full view</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QewExtensibleDialog</class>
   <extends>QDialog</extends>
   <header>/usr/include/qew5/qewextensibledialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QewSimpleDialog</class>
   <extends>QewExtensibleDialog</extends>
   <header>/usr/include/qew5/qewsimpledialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
